code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2046,3 +2046,31 @@
 	L.set_light(-1)
 
 	..()
+
+datum/reagent/elementx
+	name = "Element X"	//Sorry for the PPG refference
+	description = "Gets quickly absorbed into the bloodstream, and activates one's positive mutations."
+	color = "#000000"
+	metabolization_rate = 5 * REAGENTS_METABOLISM
+	taste_description = "empowered"
```

Comment:
This would display as 'you taste empowered'

Reply:
Intended.

Reply:
That would be better I think